collections/VideosCollection.js

```diff
@@ -0,0 +1,25 @@
+var VideosCollection = Backbone.Collection.extend({
+  url: "",
+  model: VideoModel,
+
+  parse: function (response) {
+    return response.items
+  },
+
+  //Fetchs video for search with desired searchTerm
+  searchVideos: function (searchTerm, numberOfResults = 5) {
+    //Checks to see if the number of results desired is valid. The max number of results is 50 so will just return for anything above that. Additionally, if the number of results desired is 6 this means the bottom of the page scroll event was triggered for a single video display which we also don't want to have new results returned for.
```

Comment:
Being nitpicky, but you can set your width in VSCode so that your comments will create a line break - standard is often 80 characters wide